web/packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -22,7 +22,7 @@ import { useParams } from 'react-router';
 import useAttempt from 'shared/hooks/useAttemptNext';
 
 import { ButtonState } from 'teleport/lib/tdp';
-import { useMfa } from 'teleport/lib/useMfa';
+import { useMfaTty } from 'teleport/lib/useMfa';
```

Comment:
Pretty sure this breaks per-session MFA for desktops: see #50557

Requesting changes and adding a do-not-merge label to prevent this from getting in a release.